app/uk/gov/hmrc/thirdpartyapplication/controllers/ApplicationController.scala

```diff
@@ -54,11 +54,14 @@ class ApplicationController @Inject()(val applicationService: ApplicationService
   val applicationCacheExpiry = config.fetchApplicationTtlInSecs
   val subscriptionCacheExpiry = config.fetchSubscriptionTtlInSecs
 
-  val apiGatewayUserAgents: List[String] = List("APIPlatformAuthorizer")
+  val apiGatewayUserAgent: String = "APIPlatformAuthorizer"
+  val INTERNAL_USER_AGENT = "X-GATEWAY-USER-AGENT"
+
   override implicit def hc(implicit request: RequestHeader) = {
     def header(key: String) = request.headers.get(key) map (key -> _)
+    def renamedHeader(key: String, newKey: String) = request.headers.get(key) map (newKey -> _)
 
-    val extraHeaders = List(header(LOGGED_IN_USER_NAME_HEADER), header(LOGGED_IN_USER_EMAIL_HEADER), header(SERVER_TOKEN_HEADER), header(USER_AGENT)).flatten
+    val extraHeaders = List(header(LOGGED_IN_USER_NAME_HEADER), header(LOGGED_IN_USER_EMAIL_HEADER), header(SERVER_TOKEN_HEADER), renamedHeader(USER_AGENT, INTERNAL_USER_AGENT)).flatten
```

Comment:
Can we not just strip the User-Agent header from API Gateway completely? Not sure of the value of `X-Gateway-User-Agent`?